Cargo.toml

```diff
@@ -89,9 +88,8 @@ harness = false
 [features]
 # If disabling default features, consider explicitly re-enabling the
 # "embedded-domain-resolver" feature.
-default = ["embedded-domain-resolver", "full-regex-handling", "object-pooling", "unsync-regex-caching"]
```

Comment:
We also need to disable in b-c when the a version is merged:
https://github.com/brave/brave-core/blob/master/components/brave_shields/adblock/rs/Cargo.toml#L15